ingestors/npm.go

```diff
@@ -3,16 +3,15 @@ package ingestors
 import (
 	"context"
 	"fmt"
-	"os"
 	"time"
 
 	log "github.com/sirupsen/logrus"
 
 	kivik "github.com/go-kivik/kivik/v4"
-	"github.com/go-redis/redis/v8"
 
 	_ "github.com/go-kivik/couchdb/v4"
```

Comment:
How bout this one :D

Reply:
This one's necessary for `go-kivik/kivik` to use `go-kivik/couchdb` as the driver for its Couch client.

Reply:
I also just realized it's added as an unused import mostly so that its init() function runs: https://github.com/go-kivik/couchdb/blob/b9577d0d718a7f8ca7d9b3c4156e6855f2209103/couchdb.go#L31-L33

Reply:
Yep I figured it was just so a side effect happened. WFM!

Reply:
Strange pattern tho